lib/galaxy/datatypes/sniff.py

```diff
@@ -38,6 +38,24 @@ def get_test_fname(fname):
     return full_path
 
 
+def get_test_iter(fnames):
+    """Returns test data paths separated into those that
+    should evaluate positive and negative given a list
+    of basenames that should evaluate positive.
+    """
+    pos = set()
+    neg = set()
+    path, name = os.path.split(__file__)
+    path = os.path.join(path, 'test')
```

Comment:
```suggestion
    path = os.path.join(os.path.dirname(__file__), 'test')
```

and I would probably move it outside of this function as a constant instead of recalculating it over and over. It can be reused by `get_test_fname()` as well.

Reply:
I implemented this is https://github.com/galaxyproject/galaxy/pull/12418 .. will remove this change here